java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture3/tasks/_1/Number1.java

```diff
@@ -0,0 +1,204 @@
+package com.zoolatech.lecture3.tasks._1;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/*
+Design classes that represent an orchestra, musicians and musical instruments (at least 3 different one).
+Each instrument has a name and a method to print a unique sound of the instrument.
+Each musician has a name and an associated instrument which he plays.
+It should be possible to distinguish two musicians.
+A musician should also have a method which should print his/her name and the name of an instrument.
+Then it needs to make the instrument sound. It should be possible to add/remove musicians from/to the orchestra.
+When the add/remove action is performed, the orchestra needs to print the action name + name of the musician.
+The orchestra should also have a method which commands all musicians to play the music.
+Inside of the main method, create a couple of instances of all classes and play the music.
+ */
+
+public class Number1 {
+
+    public static void main(String[] args) {
+        boolean exit = false;
+        String string;
```

Comment:
`string` can be declared inside of the switch statement

also, it would be better to give it better naming